.NET/Microsoft.Recognizers.Definitions/Italian/NumbersDefinitions.tt

```diff
@@ -0,0 +1,7 @@
+﻿<#@ template debug="true" hostspecific="true" language="C#" #>
+<#
+	this.DataFilename = @"Patterns\Italian\Italian-Numbers.yaml";
```

Comment:
I see that you have translated most json specs and that some Italian tests pass, but many fail.
I suggest you mark the current failing tests as "NotSupported": "javascript,dotnet" in the json files. This will allow the build to pass. 
As problems are fixed and new tests pass, you can remove "dotnet" from the NotSupported tag. As soon as each extractor/parser passes all its tests, we can merge little by little. No need to wait for the whole Italian recognizers to be ready at once.
This will make the review process much simpler, as well as allow others to contribute if they can.

Reply:
I also suggest start with Choice and Numbers, then Units, and leaving the DateTime fixes for last as it's the most complex recognizer.
Once you get to it, them start there with the basic types.